cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1759,6 +1759,21 @@ public static String getZKQuorum(CConfiguration cConf) {
 
     public static final String PUBLISH_ENABLED = "event.publish.enabled";
 
+    public static final String EVENTS_WRITER_PREFIX = "event.writer";
+
+    public static final String EVENTS_WRITER_EXTENSIONS_DIR = "events.writer.extensions.dir";
+
+    public static final String PROGRAM_STATUS_RECORD_EVENT_TOPIC = "program.status.record.event.topic";
+
+    public static final String EVENTS_WRITER_PUBSUB_PROJECT = "event.writer.pub_sub_event_writer.project";
```

Comment:
Pub sub constants should not be added here since pub sub is an extension and cdap is not aware of the internals of the extensions that will be loaded.